src/components/list.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import Form from './Form';
+
+export default function List() {
+  const [books] = useState([
+    {
+      id: 1,
+      title: 'Romoe and Juliet',
+      author: 'William Shakespear',
+      category: 'Non-fiction',
+      completed: false,
+
+    },
+    {
+      id: 1,
+      title: 'Art of hustling',
+      author: 'Dj sbu',
+      category: 'Non-fiction',
+      completed: false,
+
+    },
+  ]);
+
+  return (
+    <div>
+      {books.map((book) => (
+        <Form
+          key={book.id}
+          list={book}
+        />
+      ))}
+
```

Comment:
- [ ] Currently, the book items are not being rendered and the `<Form>` component appears multiple times. I think this is because the mapping logic in your current `<list>` component was iterating over the <Form> component instead of the book items. This caused the form to be displayed repeatedly instead of rendering the books individually. 
- However, if you follow my initial suggestions, and implement the `<BookList>` and `<Book>` components to correctly render the book items based on the provided data, this issue will be resolved :100: 
- Your Current UI
![image](https://github.com/techmoves/bookstores/assets/102699958/eb49ec84-cdd2-4d7c-a647-6403ed9b7979)